e2e-playwright/page-object/stories/base.storie.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2005 - 2021 Alfresco Software, Ltd. All rights reserved.
+ *
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+
+import { Page } from '@playwright/test';
+import { timeouts } from '../../utils/timeouts';
+import { ComponentTitles } from '../../models/component-titles.model';
+import { PlaywrightBase } from '../playwright-base';
+
+interface NavigateParameters { componentName: string; story: string };
+
+export class BaseStories extends PlaywrightBase {
+    private rootComponentTitle: string;
+
+    constructor(page: Page, rootComponentTitle: ComponentTitles) {
+        super(page);
+        this.rootComponentTitle = rootComponentTitle;
+    }
+
+    async navigateTo({ componentName, story }: NavigateParameters): Promise<void> {
+        await this.page.goto(`/iframe.html?id=${this.rootComponentTitle}-components-${componentName}--${story}`, {
+            waitUntil: 'networkidle',
```

Comment:
Why iframe?

Reply:
Because all of the stories are in iFrames, we can easily go into it and test the component in it. This approach I've got from this (scroll to the Example with Playwright): https://storybook.js.org/docs/react/writing-tests/importing-stories-in-tests